includes/Modules/Analytics.php

```diff
@@ -1104,7 +1104,7 @@ function ( Google_Service_Analytics_Account $account ) {
 			case 'POST:create-account-ticket':
 				// Cache the create ticket id long enough to verify it upon completion of the terms of service.
 				set_transient(
-					$self::ANALYTICS_PROVISIONING_ACCOUNT_TICKET_ID . '::' . get_current_user_id(),
+					$self::PROVISION_ACCOUNT_TICKET_ID . '::' . get_current_user_id(),
```

Comment:
`$self` is still an undefined variable here.
```suggestion
					self::PROVISION_ACCOUNT_TICKET_ID . '::' . get_current_user_id(),
```